app/models/provider_user.rb

```diff
@@ -5,6 +5,8 @@ class ProviderUser < ActiveRecord::Base
 
   before_save :downcase_email_address
 
+  audited except: [:last_signed_in_at]
```

Comment:
Does this automatically track the `has_and_belongs_to_many :providers` of the provider user?

Reply:
No, good point, I need to add that

Reply:
Not as simple as I thought - needs a separate migration and change to the many-many association - https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1194